src/test/scala/nl.knaw.dans.easy.deposit/DataFilesSpec.scala

```diff
@@ -147,6 +145,30 @@ class DataFilesSpec extends TestSupportFixture {
     ))
   }
 
+  it should " return an empty list on a new bag" in {
+    val bag = DansV0Bag
+      .empty(testDir / "testBag").getOrRecover(fail("could not create test bag", _))
+    bag.save()
+    val dataFiles = DataFiles(bag)
+    dataFiles.list() shouldBe Success(Seq.empty)
```

Comment:
But does an empty bag also **_create_** the `original` directory? If not, the two tests below are identical.

Reply:
`DansV0Bag` has no knowledge about the original folder and does not support empty directories.